composites/Plugin/SnippetPreview/components/SnippetPreview.js

```diff
@@ -133,7 +133,8 @@ export const DesktopDescription = styled.div.attrs( {
 const MobileDescription = styled( DesktopDescription )`
 	line-height: 1em;
 	max-height: 4em;
-	overflow: hidden;
+	/* Hide overflow as long as description length is being calculated */
+	overflow: ${ props => props.overflowHidden ? null : "hidden" };
```

Comment:
For better clarity, I'd expect the other way around: if there's a prop overflowHidden then set to "hidden".